src/main/java/uk/gov/companieshouse/authcodenotification/service/AuthCodeNotificationService.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.authcodenotification.service;
+
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.authcodenotification.exception.ServiceException;
+import uk.gov.companieshouse.authcodenotification.utils.ApiLogger;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+import java.util.Map;
+
+@Service
+public class AuthCodeNotificationService {
+
+    @Autowired
+    private PrivateDataRetrievalService privateDataRetrievalService;
+
+    public void sendAuthCodeEmail(String requestId, String companyNumber) throws ServiceException {
+        DataMap dataMap = new DataMap.Builder().companyNumber(companyNumber).build();
+        ApiLogger.infoContext(requestId, "Send auth code email invoked", dataMap.getLogMap());
+
+        String email = getOverseasEntityEmail(requestId, companyNumber, dataMap.getLogMap());
+
+        // 2. send email
```

Comment:
Could make this a TODO comment. At the moment it looks like something has forgotten to be implemented.